demo/iron-doc-module.html

```diff
@@ -1,6 +1,4 @@
-<script type="module" src="../default-theme.js"></script>
-<script type="module" src="../iron-doc-module.js"></script>
-<script src="../../../@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+<script src="https://unpkg.com/@webcomponents/webcomponentsjs@2/webcomponents-loader.js"></script>
 
 <!-- FIXME(polymer-modulizer):
```

Comment:
Remove FIXME comment

Reply:
Note this is a different FIXME to the one we discussed, it's https://github.com/Polymer/polymer-modulizer/issues/154.

In looking into this, I realized that the existing index.html was a better demo anyway, so I moved that here. That file *also* had the styling FIXME issue, but I found I could restore to the standard form and there were no styling issues. Not sure if that was just in this case, or if something was fixed in ShadyCSS.